aiida/cmdline/commands/cmd_code.py

```diff
@@ -88,15 +86,15 @@ def setup_code(non_interactive, **kwargs):
 
     try:
         code = code_builder.new()
-    except InputValidationError as exception:
+    except ValueError as exception:
         echo.echo_critical(f'invalid inputs: {exception}')
 
     try:
         code.store()
-        code.reveal()
-    except ValidationError as exception:
```

Comment:
Just as a comment I would maybe add a small note to the commit message saying that you also fixed this bug (the reveal being inside the with construct) and made a more open ended exception catching.

Reply:
I don't think this is necessarily bug is it?

Reply:
Yeah, I was thinking that if it was raising because of `reveal` it would actually store the thing and then say it didn't. But now I notice that since it is catching specifically the `ValidationError ` you are probably right (as `reveal` was not raising this kind of exception).